src/executor/evaluate/mod.rs

```diff
@@ -300,6 +300,59 @@ async fn evaluate_function<'a, T: 'static + Debug>(
 
             Ok(Evaluated::from(Value::Str(converted)))
         }
+        .map(Evaluated::from),
+        Function::Lpad { expr, size, fill } | Function::Rpad { expr, size, fill } => {
+            let name = if matches!(func, Function::Lpad { .. }) {
+                "LPAD"
+            } else {
+                "RPAD"
+            };
+
+            let string = match eval_to_str(name, expr).await? {
+                Nullable::Value(v) => v,
+                Nullable::Null => {
+                    return Ok(Evaluated::from(Value::Null));
+                }
+            };
+
+            let size = match eval(size).await?.try_into()? {
+                Value::I64(number) => usize::try_from(number)
+                    .map_err(|_| EvaluateError::FunctionRequiresUSizeValue(name.to_owned()))?,
+                Value::Null => {
+                    return Ok(Evaluated::from(Value::Null));
+                }
+                _ => {
+                    return Err(EvaluateError::FunctionRequiresIntegerValue(name.to_owned()).into());
+                }
+            };
+
+            let fill = match fill {
+                Some(expr) => match eval_to_str(name, expr).await? {
+                    Nullable::Value(v) => v,
+                    Nullable::Null => {
+                        return Ok(Evaluated::from(Value::Null));
+                    }
```

Comment:
There are three `Ok(Evaluated::from(Value::Null)` and a single `Err(..)` and these are not tested.
We need to have test cases for them.